index.html

```diff
@@ -1,13 +1,58 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Prim-U</title>
     <link rel="stylesheet" href="./index.css">
 </head>
+
 <body class="page">
-   
+
+    <!---  <section class="treat-yourself">
+        <h2 class="treat-yourself__heading heading">Treat Yourself With <span class="heading_span">Our Services</span></h2>
+        <ul class="treat-yourself__card-area">
+            <li class="treat-yourself__card treat-yourself__card_type_massage"><img class="treat-yourself__card-image" src="/images/massage.png"><div class="treat-yourself__card-text"><p class="treat-yourself__text">Massage</p></div></div></li>
+            <li class="treat-yourself__card treat-yourself__card_type_waxing"><img class="treat-yourself__card-image" src="/images/waxing.png"><div class="treat-yourself__card-text"><p class="treat-yourself__text">Waxing</p></div></div></li>
+            <li class="treat-yourself__card treat-yourself__card_type_make-up"><img class="treat-yourself__card-image" src="/images/makeUp.png"><div class="treat-yourself__card-text"><p class="treat-yourself__text">Make-Up</p></div></div></li>
+            <li class="treat-yourself__card treat-yourself__card_type_maincure"><img class="treat-yourself__card-image" src="/images/manicure.png"><div class="treat-yourself__card-text"><p class="treat-yourself__text">Manicure</p></div></div></li>
+            <li class="treat-yourself__card treat-yourself__card_type_hair"><img class="treat-yourself__card-image" src="/images/hair.png"><div class="treat-yourself__card-text"><p class="treat-yourself__text">Hair</p></div></div></li>
+            <li class="treat-yourself__card treat-yourself__card_type_male-grooming"><img class="treat-yourself__card-image" src="/images/maleGrooming.png"><div class="treat-yourself__card-text"><p class="treat-yourself__text">Male Grooming</p></div></div></li>
+        </ul>
+    </section> -->
+    <section class="providers">
+        <h2 class="providers__heading heading">For Those Who Provide <span class="heading_span">Beauty Services</span>
+        </h2>
+        <div class="provider__card-area">
+            <div class="provider__card provider__card_type_freelance">
+                <div class="provider__image provider__image_type_freelance">
+                    <div class="provider__card-content">
```

Comment:
Small item: when the window is size juuuuust enough such that the heading text is about to wrap, the text touches the side. What do you think about side padding so the text sits off the side of the card content container a bit?

Reply:
Great suggestion!